roles/adoption_osp_deploy/templates/adoption_vars.yaml.j2

```diff
@@ -65,6 +65,25 @@ edpm_nodes_networker:
     {% endfor %}
     {% endfor %}
 {% endif %}
+{% if edpm_node_controller_as_net | default([false]) %}
```

Comment:
This should probably default to either `[]` or `false` but i'm not sure what sets that variable? Is it meant to be a boolean or a list?

Also i'm wondering how does it interact with
https://github.com/rdo-infra/rdo-jobs/blob/49c941bf2fd6caf7d1bbda7b67f98d77b791ac82/playbooks/data_plane_adoption/deploy_tripleo_run_repo_tests.yaml#L191 

Looks like we set the same `edpm_nodes_networker` variable both here in ci-framework and also in rdo-jobs. What gets used in the end? Sometimes i get confused by how our config is spread across so many repos which are apparently setting the same things... :)